packages/mobile/src/app/App.tsx

```diff
@@ -87,6 +87,7 @@ export class App extends React.Component<Props> {
       deviceWidth: width,
       reactLoadDuration,
       appLoadDuration,
+      language: i18n.language,
```

Comment:
Sometimes `i18n` has weird initialization issues that cause random crashes and stuff. Does pulling language in this way have the same net effect as pulling from the Redux store?

Reply:
I though about this, `i18n.language` has been null at times, yes, but that is most likely when the app is opened for the first time and this code is run before the store initialization, so language is probably going to be empty there as well. I'm not sure which one is best, maybe I can add one as fallback for the other

Reply:
Added a fallback on the store value. On my tests, both are not null, but there might be edge cases, so I think having both will be probably good.